compiler/rustc_mir_transform/src/check_unnecessary_transmutes.rs

```diff
@@ -1,3 +1,4 @@
+use rustc_hir::def::DefKind::*;
```

Comment:
Remove this, but also in general please don't glob import things unless there's a really good reason